experimental/rest_client.go

```diff
@@ -0,0 +1,90 @@
+package experimental
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"path"
+	"time"
+)
+
+// Client interface
+type Client interface {
+	Fetch(ctx context.Context, uriPath, uriQuery string) ([]byte, error)
+	Get(ctx context.Context, uriPath, uriQuery string) (*http.Response, error)
```

Comment:
Shall we put the documentation comments on the interface methods instead, since they're what SDK clients will see?